client/src/components/navigation/Navbar.js

```diff
@@ -1,30 +1,47 @@
-import React from 'react';
+import React from "react";
```

Comment:
Super nitpicky, but we should standardize the use of single quote since we already started.